### PR TITLE
Align credentials stores part 17

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5274)

## Description
Change the type of credentials, from type 4 to type 9.

## Minimum version of Cortex XSOAR
- [x] 6.5.0

## Does it break backward compatibility?
  - [x] No

### CODE REVIEW

**Comment by maimorag** [comment]
The build fails since there is no instance [IN140] for:
- Pipl
- Twilio

File: Packs/Twilio/Integrations/Twilio/Twilio.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 var sendRequest = function(method, url, body) {
+    var username = (params.credentials_token!== undefined)? params.credentials_token.identifier : params.sid;
```
Comment (Shellyber) [inline Packs/Twilio/Integrations/Twilio/Twilio.js:2]: Change the variable's names to the right ones - token, account_sid

File: Packs/Twilio/Integrations/Twilio/Twilio.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+    var password = (params.credentials_token!== undefined)? params.credentials_token.password : params.token;
+    if (!(username && password)){
+        throw('API key and Auth token must be provided.');
```
Comment (Shellyber) [inline Packs/Twilio/Integrations/Twilio/Twilio.js:5]:
```suggestion
        throw('Account SID and Auth token must be provided.');
```

File: Packs/Twilio/ReleaseNotes/1_0_5.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Twilio
+Added the *API key* and *Auth token* integration parameters to support credentials fetching object.
```
Comment (Shellyber) [inline Packs/Twilio/ReleaseNotes/1_0_5.md:5]:
```suggestion
Added the *Account SID* and *Auth token* integration parameters to support credentials fetching object.
```

**Review by Shellyber — APPROVED** [review]

File: Packs/Pipl/Integrations/Pipl/Pipl.js [hunk 4]
```diff
@@ -10,6 +10,9 @@ var sendRequest = function(args) {
 
     var queryArgs = {};
     var argKeys = Object.keys(args);
+    var key =  params.credentials_api_key !== null ? params.credentials_api_key.password : params.key;
```
Comment (dantavori) [inline Packs/Pipl/Integrations/Pipl/Pipl.js:13]:
```suggestion
    var key =  params.credentials_api_key || params.key;
```

File: Packs/Pipl/Integrations/Pipl/Pipl.js [hunk 5]
```diff
@@ -10,6 +10,9 @@ var sendRequest = function(args) {
...
+    if (key !== null)
```
Comment (dantavori) [inline Packs/Pipl/Integrations/Pipl/Pipl.js:14]:
```suggestion
    if (!key)
```

File: Packs/Twilio/Integrations/Twilio/Twilio.js [hunk 6]
```diff
@@ -1,12 +1,17 @@
 var sendRequest = function(method, url, body) {
+    var account_sid = (params.credentials_token!== undefined)? params.credentials_token.identifier : params.sid;
+    var auth_token = (params.credentials_token!== undefined)? params.credentials_token.password : params.token;
+    if (!(account_sid && auth_token)){
+        throw('Account SID and Auth token must be provided.');
+    }
```
Comment (dantavori) [inline Packs/Twilio/Integrations/Twilio/Twilio.js:6]:
```suggestion
    var account_sid = params.credentials_token.identifier || params.sid;
    var auth_token = params.credentials_token || params.token;
    if (!(account_sid && auth_token)) {
        throw('Account SID and Auth token must be provided.');
    }
```

**Review by dantavori — CHANGES_REQUESTED** [review]

File: Packs/Pipl/Integrations/Pipl/Pipl.js [hunk 7]
```diff
@@ -10,6 +10,10 @@ var sendRequest = function(args) {
 
     var queryArgs = {};
     var argKeys = Object.keys(args);
+    var key =  (!params.credentials_api_key) ? params.credentials_api_key.password : params.key;
```
Comment (dantavori) [inline Packs/Pipl/Integrations/Pipl/Pipl.js:13]:
```suggestion
    var key =  params.credentials_api_key ? params.credentials_api_key.password : params.key;
```

File: Packs/Twilio/Integrations/Twilio/Twilio.js [hunk 8]
```diff
@@ -1,12 +1,18 @@
 var sendRequest = function(method, url, body) {
+    var account_sid = (!params.credentials_token)? params.credentials_token.identifier : params.sid;
+    var auth_token = (!params.credentials_token)? params.credentials_token.password : params.token;
```
Comment (dantavori) [inline Packs/Twilio/Integrations/Twilio/Twilio.js:3]:
```suggestion
    var account_sid = params.credentials_token ? params.credentials_token.identifier : params.sid;
    var auth_token = params.credentials_token ? params.credentials_token.password : params.token;
```

**Pull request merged**